src/components/pill/pill.scss

```diff
@@ -3,14 +3,38 @@
 .carbon-pill {
   border-radius: 10px;
   display: inline-block;
-  font-size: 11px;
+  font-size: 12px;
   font-weight: 700;
-  line-height: 12px;
   letter-spacing: 1px;
   padding: 2px 7px;
   text-align: center;
 }
 
+.carbon-pill_with-icon {
+  font-size: 14px;
+  line-height: 14px;
+}
+
+.carbon-pill__delete-icon {
+  margin-left: 5px;
+  margin-right: -5px;
+}
+.carbon-pill__delete-icon:before {
+  font: {
+    size: 10px;
+  }
+  line-height: 10px;
+  margin-left: 2px;
+  margin-right: 2px;
+}
+
+.carbon-pill__delete-icon:hover {
```

Comment:
Needs `:focus` styles too for keyboard users.

Reply:
Thanks, I removed it when UX told me they wanted to go in a different direction than having the Icon tabbable, but obviously I should of left the css the same. I've changed it back to have `:focus` styles